src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Trustly/TrustlyPaymentsResource.cs

```diff
@@ -0,0 +1,33 @@
+﻿using SwedbankPay.Sdk.Common;
+using SwedbankPay.Sdk.Extensions;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Trustly
+{
+    public class TrustlyPaymentsResource : ResourceBase, ITrustlyResource
+    {
+        public TrustlyPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public async Task<ITrustlyPaymentResponse> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)